modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductMedia.testcase

```diff
@@ -85,12 +85,73 @@ definition {
 	}
 
 	@description = "This is a test for COMMERCE-6031. EditAnAttachment"
-	@ignore = "true"
 	@priority = "3"
 	test EditAnAttachment {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6031 EditAnAttachment
+		ApplicationsMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Control Panel",
+			portlet = "Custom Fields");
+
+		CustomFields.addCP(
+			customFieldName = "Custom Field",
+			customFieldType = "Text",
+			customInputDataType = "Text",
+			resourceName = "Commerce Product Attachment");
+
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Catalogs");
+
+		Click(locator1 = "Button#PLUS");
+
+		CommerceEntry.addNewCatalogsEntry(
+			catalogCurrency = "US Dollar",
+			catalogLanguage = "English (United States)",
+			catalogName = "Master");
+
+		AssertClickNoError(
+			locator1 = "Button#SUBMIT",
+			value1 = "Submit");
+
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Products");
+
+		CPCommerceCatalog.newProductsEntry(
+			productCatalog = "Master",
+			productName = "Simple T-Shirt",
+			productType = "Simple");
+
+		CommerceEntry.gotoMenuTab(menuTab = "Media");
+
+		CommerceProductImages.addAttachmentImages(imageTitles = "Black");
+
+		Click(
+			key_value = "Commerce_Black.jpg",
+			locator1 = "CommerceEntry#BODY_VERTICAL_ELLIPSIS");
+
+		MenuItem.click(menuItem = "Edit");
+
+		SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+		Type(
+			locator1 = "TextInput#TITLE",
+			value1 = "Commerce_Black_Edit");
+
+		Button.clickSaveAsDraft();
```

Comment:
I think this can be removed because it's a duplicate

Reply:
Done.